scripts/demangle.py

```diff
@@ -2,23 +2,38 @@
 import json
 import glob
 import re
+import docker
 import ctypes
+import cpp_demangle
 
-json_path = '../dump/augeas'
-proj_path = '../oss-fuzz/projects/augeas'
-file_names = [f for f in os.listdir(json_path) if f.endswith('.json') and os.path.isfile(json_path+'/'+f)]
-print('Looking for json file: ',file_names)
-def extract_func_code(func_name, file_path):
-    with open(file_path, 'r') as file:
-        code_content = file.read()
+proj_name = 'tmux'
+json_path = '../dump/'+proj_name
+json_file_names = [f for f in os.listdir(json_path) if f.endswith('.json')]
+print('Looking for json file: ',json_file_names)
+cnt=0
+
+def extract_func_code(func_name, code_content, if_c_code=True):
     # init regular expression
-    match_func_init = re.search(".*"+func_name,code_content)
+    try:
+        # Transliteration special characters
+        ori_func_name=func_name
+        pattern = r"[\[\].,{}()\W_]"
+        # Replace special characters with \ ahead of themselves
+        func_name = re.sub(pattern, r"\\\g<0>", func_name)
+        # If C code, there is no parphsis in the func_name, so it need to manually add it
+        if if_c_code:
+            match_func_init = re.search(".*"+func_name+'[^;]*\).*\{\n',code_content)
+        else:
+            match_func_init = re.search(".*"+func_name+'[^;]*\n',code_content)
+    except:
```

Comment:
Add the `Exception` you are expecting. 
If you only `except`, you may not abort this code using  Ctrl+C. (Because KeyboardException is also caught by this)